spec.yaml

```diff
@@ -93,7 +93,11 @@ paths:
           "general": {
             "installation_name": "shire",
             "provider": "aws",
-            "datacenter": "eu-central-1"
+            "datacenter": "eu-central-1",
```

Comment:
`datacenter` field seems to have been there already earlier, but it somehow struck me now that this is wrong. Each AZ is a datacenter and `datacenter` here should most probably be a `region`. Probably worth handling separately, but just mentioning here as I happened to notice this.

Reply:
:+1: but lets do this separately.